docs/static/resources/openapi.json

```diff
@@ -561,7 +561,7 @@
       "CacheInvalidationRequestSchema": {
         "properties": {
           "datasource_uids": {
-            "description": "The uid of the dataset/datasource this new chart will use. A complete datasource identification needs `datasouce_uid` ",
+            "description": "The uid of the dataset/datasource this new chart will use. A complete datasource identification needs `datasource_uid` ",
```

Comment:
this needs to be changed here: https://github.com/apache/superset/blob/master/superset/charts/schemas.py#L109 and then regenerate the OAS, the spec should not be edited directly

Reply:
For reference, that *was* changed in this PR by the same commit.

However, in the interest of not dealing w/ this mess at this time, I'm dropping all commits that touch this file. We can deal w/ it in a distinct PR, or perhaps someone else can.